CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
+* Modify response[Contains|NotContains] to use strpos vs preg_match
+* Fix compatibility with PHP 7.2
+* Allow Symfony 4.
+* Optimize the splitting of XPath unions
+* Allow pipes (|) in attributes by implementing better XPath parsing
+* Auto-start session only upon 1st "->visit(...)" method call
+
+Testsuite:
+
+* Don't install PHPUnit 8.0
+* Fix PHP 5.4 and PHP 5.5 builds
+* Objects with __toString() can be asserted.
+* Add testing on PHP 7.2
+* Add compatibility with PHPUnit 6
+* Extracted driver test suite into separate repository.
+
+Misc:
+
+* Rename boolean type in lowercase
+* Suggest chrome-mink-driver as fast JS-enabled alternative to selenium
+* Use ! operator instead of === false
+* Improved variable names and simplified quote logic
+* Changed method visibility to private and variable name to camle case
```

Comment:
Please remove this line.

---

This is an internal change, that shouldn't be affecting developer workflow.